### PR TITLE
Pub 348 test

### PR DESCRIPTION
**Pull request by Matthew-Summerbell**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by subrotomukherjee — APPROVED** [review]

File: src/acceptanceTest/java/uk/gov/hmcts/futurehearings/pip/acceptance/publications/GETPublicationsTest.java [hunk 1]
```diff
@@ -1,19 +1,13 @@
 package uk.gov.hmcts.futurehearings.pip.acceptance.publications;
```
Comment (ShufenZ) [inline src/acceptanceTest/java/uk/gov/hmcts/futurehearings/pip/acceptance/publications/GETPublicationsTest.java:1]: Could you please explain the reason(s) to have this test file with no test in it?

File: src/acceptanceTest/java/uk/gov/hmcts/futurehearings/pip/acceptance/publications/POSTPublicationsTest.java [hunk 2]
```diff
@@ -1,14 +1,10 @@
 package uk.gov.hmcts.futurehearings.pip.acceptance.publications;
```
Comment (ShufenZ) [inline src/acceptanceTest/java/uk/gov/hmcts/futurehearings/pip/acceptance/publications/POSTPublicationsTest.java:1]: Could you please explain the reason(s) to have this test file with no test in it?

**Review by ShufenZ — CHANGES_REQUESTED** [review]
The comments here are in conjunction of two files below:

src/acceptanceTest/java/uk/gov/hmcts/futurehearings/pip/acceptance/common/rest/RestClientTemplate.java

and

src/functionalTest/java/uk/gov/hmcts/futurehearings/pip/functional/common/rest/RestClientTemplate.java

These two files above are responsible for the same purposes of getting response from the Endpoints under tests for different operation calls.

Why they are inconsistent at the following aspects?
1. The ordering and naming of the parameters
2. The first one has "HttpStatus expectedHttpStatus" but the second one dose not

Please explain the reason(s) that they should be implemented separately and differently. Otherwise, please make only one of them to be share by all applicable test types (e.g. acceptance and functional etc.).

File: src/acceptanceTest/java/uk/gov/hmcts/futurehearings/pip/acceptance/publications/GETPublicationsTest.java (hunk 1 above)
Comment (Matthew-Summerbell) [reply src/acceptanceTest/java/uk/gov/hmcts/futurehearings/pip/acceptance/publications/GETPublicationsTest.java:1]: These were the template files, just removed the tests can remove the file

File: src/acceptanceTest/java/uk/gov/hmcts/futurehearings/pip/acceptance/publications/POSTPublicationsTest.java (hunk 2 above)
Comment (Matthew-Summerbell) [reply src/acceptanceTest/java/uk/gov/hmcts/futurehearings/pip/acceptance/publications/POSTPublicationsTest.java:1]: as above

File: src/acceptanceTest/java/uk/gov/hmcts/futurehearings/pip/acceptance/publications/GETPublicationsTest.java (hunk 1 above)
Comment (ShufenZ) [reply src/acceptanceTest/java/uk/gov/hmcts/futurehearings/pip/acceptance/publications/GETPublicationsTest.java:1]: > These were the template files, just removed the tests can remove the file

Did you mean that you cannot remove the file as it to be used as template?

**Review by ShufenZ — APPROVED** [review]

**Review by subrotomukherjee — APPROVED** [review]

**Pull request merged**